dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -141,6 +148,20 @@ export const SignInGateSelector = ({
 	idUrl = 'https://profile.theguardian.com',
 }: Props) => {
 	const isSignedIn = !!getCookie({ name: 'GU_U', shouldMemoize: true });
+	// START: Checkout Complete Personalisation
+	const switches = window.guardian.config.switches;
```

Comment:
Maybe we could use the [Gu switch reading lib](https://github.com/guardian/csnx/tree/main/libs/%40guardian/libs/src/switches) here? It false back to fetching from [here ](https://www.theguardian.com/switches.json)if the window object not avail.